keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -111,32 +112,22 @@ open class SqlResourceRepository(
     }
   }
 
-  fun getUidByApplication(application: String): List<String> {
-    return sqlRetry.withRetry(READ) {
-      jooq
-        .select(RESOURCE.UID)
-        .from(RESOURCE)
-        .where(RESOURCE.APPLICATION.eq(application))
-        .fetch(RESOURCE.UID)
-    }
-  }
-
-  override fun getSummaryByApplication(application: String): List<ResourceSummary> {
-    val resources: List<Resource<*>> = sqlRetry.withRetry(READ) {
+  override fun getResourceSummaries(deliveryConfig: DeliveryConfig): List<ResourceSummary> {
+    val resourceSummaries: List<ResourceSummary> = sqlRetry.withRetry(READ) {
       jooq
         .select(RESOURCE.KIND, RESOURCE.METADATA, RESOURCE.SPEC)
         .from(RESOURCE)
-        .where(RESOURCE.APPLICATION.eq(application))
+        .where(RESOURCE.APPLICATION.eq(deliveryConfig.application))
```

Comment:
Side-comment: this made me realize that we currently do not have a reference from the `resource` table to `delivery_config`, which would be required to allow multiple delivery configs per application and still have the children associated with the corresponding parent delivery config. #912

Reply:
Theres a delivery config -> env reference, and an env -> resource reference. That should accomplish what we need.

Reply:
Good point. So why do we even need that `application` column in the `resource` table? Was this the only use?

Reply:
It's probably legacy from when we only had resources and hadn't built delivery configs yet. If we're not using it to optimize any queries we could drop it (separately from this change).